pyproject.toml

```diff
@@ -221,25 +219,33 @@ type_incorrect_long = "The filename of the changelog entry must be of the form `
 
 [tool.ruff]
 target-version = "py39"
-exclude = [
-    ".direnv",
+select = [
+  "C4", # flake8-comprehensions
+  "E", # pycodestyle errors
+  "F", # pyflakes
+  "ISC", # flake8-implicit-str-concat
+  "PIE", # flake8-pie
+  "RUF", # ruff specific rules
+  "TRY", # tryceratops
+  "UP", # pyupgrade
+  "W", # pycodestyle warnings
+]
+ignore = [
+  "E501", # line-too-long
+  "RUF001", # ambiguous-unicode-character-string
+  "RUF002", # ambiguous-unicode-character-docstring
+  "RUF003", # ambiguous-unicode-character-comment
+  "RUF100", # unused-noqa (remove if we drop flake8 completely)
+  "TRY003",
+  "TRY301",
+  "UP025", # unicode-kind-prefix
+]
+extend-exclude = [
```

Comment:
Using `extend-exclude` instead of `exclude` lets us avoid having to re-specify files/paths that are already ignored by default.